as-modules/embedded/build.xml

```diff
@@ -63,7 +63,7 @@
             <maven-resource group="org.infinispan" artifact="infinispan-cachestore-jpa" />
         </module-def>
 
-        <module-def name="org.infinispan.cdi" slot="${infinispan.slot}">
+        <module-def name="org.infinispan.cdi.embedded" slot="${infinispan.slot}">
```

Comment:
@tristantarrant I have added a commit that creates an alias named org.infinispan.cdi targetting org.infinispan.cdi.embedded.